Formula/vowpal-wabbit.rb

```diff
@@ -1,10 +1,7 @@
 class VowpalWabbit < Formula
   desc "Online learning algorithm"
   homepage "https://github.com/VowpalWabbit/vowpal_wabbit"
-  # pull from git tag to get submodules
-  url "https://github.com/VowpalWabbit/vowpal_wabbit.git",
-      tag:      "9.0.1",
-      revision: "57e8ff252a8b790d14acdb7867ba25b55f3b6d26"
+  url "https://github.com/VowpalWabbit/vowpal_wabbit/archive/9.0.1.tar.gz",
```

Comment:
```suggestion
  url "https://github.com/VowpalWabbit/vowpal_wabbit/archive/9.0.1.tar.gz"
  sha256 "50c0a766c3f0f4a4ba29ac1db8949ee352e0d7e4f2df819fb410f8a34e3ea051"
```
We need a checksum too if we're using a tarball.

Reply:
Thanks for your help